packages/apps/explorer/src/components/layout/styles.css.ts

```diff
@@ -16,5 +16,6 @@ export const layoutWrapperClass = style({
 });
 
 export const contentClass = style({
+  height: '64px',
```

Comment:
need to use a token for this

Reply:
Isa mentioned that the header height isn't tokenized...